LocalWiki.php

```diff
@@ -98,6 +98,9 @@ function onInactiveSiteNoticeAfter( &$siteNotice, $skin ) {
  	];
 }
 
+// $wgRCMaxAge
+$wgRCMaxAge = ceil($wgRCMaxAge/86400);  // Convert to seconds
```

Comment:
Won't work. ManageWiki would override.

Reply:
oh, so would `$wgRCMaxAge = ceil($wmgRCMaxAge/86400);` and make the managewiki one, wmg?

Reply:
yeh.

Reply:
done